save-demo/src/main/kotlin/com/saveourtool/save/demo/runners/cli/DiktatCliRunner.kt

```diff
@@ -32,57 +32,41 @@ import kotlin.io.path.*
 @Component
 class DiktatCliRunner(
     private val toolStorage: ToolStorage,
-) : CliRunner<DiktatDemoAdditionalParams, DiktatDemoResult> {
-    override fun getExecutable(workingDir: Path, params: DiktatDemoAdditionalParams): Path {
-        val toolName = params.tool.name.lowercase()
-        val version = if (params.tool == DiktatDemoTool.DIKTAT) {
-            DIKTAT_VERSION
-        } else {
-            KTLINT_VERSION
+) : CliRunner<DiktatDemoAdditionalParams, ToolKey, DiktatDemoResult> {
+    private fun getRunCommandForDiktat(
+        workingDir: Path,
+        testPath: Path,
+        outputPath: Path,
+        configPath: Path?,
+        params: DiktatDemoAdditionalParams,
+    ): String = buildString {
+        // TODO: this information should not be hardcoded but stored in database
+        val ktlintExecutable = getExecutable(workingDir, DiktatDemoTool.KTLINT.toToolKey("ktlint"))
+        val diktatExecutable = getExecutable(workingDir, DiktatDemoTool.DIKTAT.toToolKey("diktat-1.2.3.jar"))
+        append(ktlintExecutable)
+        append(" -R $diktatExecutable ")
+        append(" --disabled_rules=standard")
+        append(" --reporter=plain,output=$outputPath ")
+        configPath?.let {
+            append(" --config=$it ")
```

Comment:
Is it supposed to be diktat config? Diktat doesn't accept config from an argument, because ktlint doesn't give access to the arguments. You can use a JVM property or rely on a specific location of a file

Reply:
Ok, thanks, will fix it

Reply:
Removed it as config file is saved in working dir. Let's try to rely on a specific location (and maybe cover it with tests)